src/pixel-tester.ts

```diff
@@ -0,0 +1,107 @@
+import * as ava from 'ava';
+import * as fs from 'fs-extra';
+import * as jimp from 'jimp';
+import * as path from 'path';
+import {PNG} from 'pngjs';
+import * as puppeteer from 'puppeteer';
+
+// tslint:disable-next-line:no-require-imports
+import pixelmatch = require('pixelmatch');
```

Comment:
maybe add a comment on why this is require instead an import

Reply:
Done.